Source/WebCore/Modules/webdatabase/Database.cpp

```diff
@@ -92,19 +92,13 @@ namespace WebCore {
 // (and un-returned) databases instances.
 
 static const char versionKey[] = "WebKitDatabaseVersionKey";
-static const char unqualifiedInfoTableName[] = "__WebKitDatabaseInfoTable__";
+static constexpr auto unqualifiedInfoTableName = "__WebKitDatabaseInfoTable__"_s;
 const unsigned long long quotaIncreaseSize = 5 * 1024 * 1024;
 
-static const char* fullyQualifiedInfoTableName()
+static const String& fullyQualifiedInfoTableName()
 {
-    static const char qualifier[] = "main.";
-    static char qualifiedName[sizeof(qualifier) + sizeof(unqualifiedInfoTableName) - 1];
-
-    static std::once_flag onceFlag;
-    std::call_once(onceFlag, [] {
-        snprintf(qualifiedName, sizeof(qualifiedName), "%s%s", qualifier, unqualifiedInfoTableName);
-    });
-
+    static constexpr auto qualifier = "main."_s;
```

Comment:
Why not just write `"main."_s` below? Why keep this as a named constant?

Reply:
Will drop.